README.md

```diff
@@ -65,9 +71,9 @@ We use [Docker Compose][docker-compose] to orchestrate these services. As such,
 
```

Comment:
Even though the sandbox link UI shows the username / password / 2fa for linking, I think people still miss it and it would be good to mention in the readme